src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/RegisterActionAnalyzer.cs

```diff
@@ -90,7 +112,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
             get
             {
                 return ImmutableArray.Create(
-                    MissingKindArgumentRule,
+                    MissingSymbolKindArgumentRule,
                     UnsupportedSymbolKindArgumentRule,
                     InvalidSyntaxKindTypeArgumentRule,
                     StartActionWithNoRegisteredActionsRule,
```

Comment:
Well, the analyzer driver just cares about unique diagnostic IDs for supported rules, but you are correct, we should just return all three rules here in case the driver implementation changes in future.